src/components/audio_source.rs

```diff
@@ -36,6 +36,30 @@ impl AudioSource {
                     .unwrap();
             });
     }
+
+    pub fn set_parameter(&self, name: &'static str, value: f32, ignore_seek_speed: bool) {
+        self.event_instance
+            .set_parameter_by_name(name, value, ignore_seek_speed)
+            .expect("Could not set parameter.");
+    }
+
+    pub fn set_parameter_labeled(
+        &self,
+        name: &'static str,
+        label: &'static str,
+        ignore_seek_speed: bool,
+    ) {
+        self.event_instance
+            .set_parameter_by_name_with_label(name, label, ignore_seek_speed)
+            .expect("Could not set parameter by label.");
+    }
+
+    /// Returns (value, final_value)
+    /// Value: "Parameter value as set from the public API."
+    /// Final value: "the final value of the parameter after applying adjustments due to automation, modulation, seek speed, and parameter velocity to value."
+    pub fn get_parameter_value(&self, name: &'static str) -> (f32, f32) {
+        self.event_instance.get_parameter_by_name(name).unwrap()
+    }
```

Comment:
This function is unused.

Reply:
It's up to the user to decide if they want to use it, or do you mean to say you want all functions we expose to be in the examples?

Reply:
It's a wrapper of a wrapper (libfmod). Maybe we should just make event_instance pub?

Reply:
Maybe we should remove this until we have a more robust solution. I would like to have no string mappings, but instead collect all events by ID at build time and generate an enum the developer can use. That way it is "type-safe", as only valid events can be used without compiler error and there is less startup / runtime overhead in the application.

Reply:
Agreed, I've changed it now to having event_instance pub